### ISSUE TITLE
再構成誤差を平均二乗誤差にしない理由

### ISSUE THREAD

**Issue by kazucmpt** [kose/chainer-mnist-VAE#1]
VAEのロス関数について質問します。
net.pyのl45についてです。

rec_loss += F.bernoulli_nll(x, self.decode(z, sigmoid=False)) \ (self.k * batchsize)

は再構成誤差なので、入力と出力がどれくらい違うかを表している訳ですが、単純に考えれば最小二乗誤差を使うのが最も素朴かと思われますが

rec_loss += F.mean_squared_error(x, self.decode(z)) / k

にしない理由は何でしょうか。また実際にl45を書き換えて、最小二乗誤差を使うと学習がうまくいきません。なぜでしょうか。
